add_on_usage.go

```diff
@@ -0,0 +1,126 @@
+package recurly
+
+import (
+	"context"
+	"encoding/xml"
+	"fmt"
+	"net/http"
+)
+
+// AddOnUsageService manages the interactions for add-on usages.
+type AddOnUsageService interface {
+	// List returns a pager to paginate usages for add-on in subscription. PagerOptions are used to
+	// optionally filter the results.
+	//
+	// https://dev.recurly.com/docs/list-add-ons-usage
+	List(subUUID, addOnCode string, opts *PagerOptions) Pager
```

Comment:
Let's rename `subUUID` to just `uuid` everywhere (to match the naming convention for subscription uuids throughout the rest of the repo).

Reply:
Okey - changed